databricks/koalas/tests/test_stats.py

```diff
@@ -98,12 +98,22 @@ def test_abs(self):
                 "B": [1.0, -2, 3, -4, 5],
                 "C": [-6.0, -7, -8, -9, 10],
                 "D": ["a", "b", "c", "d", "e"],
+                "E": [True, False, False, True, True],
             }
         )
         kdf = ks.from_pandas(pdf)
         self.assert_eq(kdf.A.abs(), pdf.A.abs())
         self.assert_eq(kdf.B.abs(), pdf.B.abs())
+        self.assert_eq(kdf.E.abs(), pdf.E.abs())
+        # pandas' bug?
+        # self.assert_eq(kdf[["B", "C", "E"]].abs(), pdf[["B", "C", "E"]].abs())
```

Comment:
The pandas result is:

```py
>>> pdf[["B", "C", "E"]].abs()
   B   C  E
0  1   6  1
1  2   7  0
2  3   8  0
3  4   9  1
4  5  10  1
```

whereas calculating separately:

```py
>>> pdf[["B", "C"]].abs()
     B     C
0  1.0   6.0
1  2.0   7.0
2  3.0   8.0
3  4.0   9.0
4  5.0  10.0
>>> pdf[["E"]].abs()
       E
0   True
1  False
2  False
3   True
4   True
```

I believe this is a pandas bug.

The current Koalas behavior:

```py
>>> ks.from_pandas(pdf)[["B", "C", "E"]].abs()
     B     C      E
0  1.0   6.0   True
1  2.0   7.0  False
2  3.0   8.0  False
3  4.0   9.0   True
4  5.0  10.0   True
```

Reply:
Seems pandas bug to me, too. Let me report this to pandas dev.